types/block.go

```diff
@@ -354,6 +354,8 @@ type Header struct {
 	// hashes from the app output from the prev block
 	ValidatorsHash     cmtbytes.HexBytes `json:"validators_hash"`      // validators for the current block
 	NextValidatorsHash cmtbytes.HexBytes `json:"next_validators_hash"` // validators for the next block
+	ValidatorsSSZ      cmtbytes.HexBytes `json:"validators_ssz"`       // validators ssz
+	NextValidatorsSSZ  cmtbytes.HexBytes `json:"next_validators_ssz"`  // next_validators ssz
```

Comment:
Nit: I think for clarity and consistency this rather should be called (Next)ValidatorsSSZHash. Otherwise this sounds like the whole serialized struct bytes are included in the header.